.babelrc

```diff
@@ -0,0 +1,11 @@
+{
+  "presets": [
+    [
+      "@parcel/babel-preset-env",
```

Comment:
J'ai ça comme message dans la console :

```
@parcel/transformer-babel: Parcel includes transpilation by default. Babel config .babelrc contains only redundant presets. Deleting it may
significantly improve build performance.

  /Users/david/Sites/mesconseilscovid/.babelrc:4:7
    3 |     [
  > 4 |       "@parcel/babel-preset-env",
  >   |       ^^^^^^^^^^^^^^^^^^^^^^^^^^
    5 |       {
    6 |         "corejs": 3,

  💡 Delete .babelrc
  📝 Learn more: https://parceljs.org/languages/javascript/#default-presets
```